src/Object/Api/Mastodon/Account.php

```diff
@@ -112,10 +113,10 @@ public function __construct(BaseURL $baseUrl, array $publicContact, Fields $fiel
 
 		$this->note            = BBCode::convert($publicContact['about'], false);
 		$this->url             = $publicContact['url'];
-		$this->avatar          = $userContact['avatar'] ?? $publicContact['avatar'];
-		$this->avatar_static   = $userContact['avatar'] ?? $publicContact['avatar'];
-		$this->header          = ($userContact['header'] ?? $publicContact['header']) ?? '';
-		$this->header_static   = ($userContact['header'] ?? $publicContact['header']) ?? '';
+		$this->avatar          = ($userContact['photo'] ?? $publicContact['photo']) ?: Proxy::proxifyUrl($userContact['avatar'] ?? $publicContact['avatar']);
+		$this->avatar_static   = $this->avatar;
+		$this->header          = Proxy::proxifyUrl(($userContact['header'] ?? $publicContact['header']) ?? '');
```

Comment:
No need for parentheses with the null coalescing operator `??` the way we do with the ternary operator `?:`.
```suggestion
		$this->header          = Proxy::proxifyUrl($userContact['header'] ?? $publicContact['header'] ?? '');
```